trip_reservations/src/main/kotlin/pl/edu/pg/rsww/tripreservations/TripReservationOrchestrator.kt

```diff
@@ -98,6 +95,9 @@ public class TripReservationOrchestrator(
             println("C2")
             return
         }
+        if (event.outcome == 0L) {
+            revertSaga()
```

Comment:
We don't want to start a next stage (which `sendBookTransport()` at the end of this function does) when triggering a saga revert.